src/main/webapp/app/therapeuticareas/views/therapeuticareas.view.html

```diff
@@ -26,16 +26,16 @@
 
                 <!-- This <li> will only add the 'active' class if 'therapeuticAreas.detail' or its descendants are active
                      AND if it is the link for the active contact (aka contactId) -->
-                <li data-ng-repeat="therapeuticArea in therapeuticAreas " ui-sref-active="active">
+                <li data-ng-repeat="therapeuticArea in therapeuticAreas " data-ui-sref-active="active">
 
-                    <!-- Here's a ui-sref that is also providing necessary parameters -->
-                    <a ui-sref="therapeuticAreas.detail({therapeuticAreaId:therapeuticArea.id})">{{therapeuticArea.title}}</a>
+                    <!-- Here's a data-ui-sref that is also providing necessary parameters -->
+                    <a data-ui-sref="therapeuticAreas.detail({therapeuticAreaId:therapeuticArea.id})">{{therapeuticArea.title}}</a>
```

Comment:
could we see about putting an id on the anchors? The testers will more than likely want to use the existing primed data for the likes of trialInfo/trialSite tests as opposed to creating their own each time.